src/pykeen/models/base.py

```diff
@@ -751,30 +751,35 @@ class EntityRelationEmbeddingModel(_OldAbstractModel, ABC, autoreset=False):
     """A base module for KGE models that have different embeddings for entities and relations."""
 
     #: Primary embeddings for entities
-    entity_embeddings: Embedding
+    entity_embeddings: Representation
+
     #: Primary embeddings for relations
-    relation_embeddings: Embedding
+    relation_embeddings: Representation
 
     def __init__(
         self,
         *,
         triples_factory: CoreTriplesFactory,
-        entity_representations: EmbeddingSpecification,
-        relation_representations: EmbeddingSpecification,
+        entity_representations: HintOrType[Representation] = None,
+        entity_representation_kwargs: OptionalKwargs = None,
```

Comment:
shouldn't the plural match so this is 
```suggestion
        entity_representations_kwargs: OptionalKwargs = None,
```

Reply:
https://github.com/pykeen/pykeen/commit/52bd6422ea54d9c1b08eed46592524ffd2cdec5d